src/components/lab-callout/interactive-lab-callout.module.css

```
@@ -0,0 +1,61 @@
+.interactiveCallout {
+  align-items: center;
+  background-color: var(--token-color-foreground-strong);
+  border-radius: 6px;
+  padding: 16px;
+
+  @media (--medium-up) {
+    display: flex;
+    gap: 32px;
+  }
+}
+
+.title {
+  composes: hds-typography-body-200 from global;
+  color: var(--token-color-surface-primary);
+  font-weight: var(--token-typography-font-weight-semibold);
+
+  /**
+   * Note: !important needed, as generic <h2> styles leak from DevDotContent.
+   * Task to fix:https://app.asana.com/0/1201987349274776/1201653123138729/f
+   */
+  margin: 0 !important;
+}
+
+.description {
+  composes: hds-typography-body-100 from global;
+
+  /**
+   * TODO: raise line-height rounding issue with design-systems
+   * TLDR; line-height defined in relative-to-font-size units often
+   * ends up with weird rounding. Re-defined in rems here to fix that.
+   * (line-height is already indirectly rems, since font-size is rems).
+   * Details in Asana task:
+   * https://app.asana.com/0/1201987349274776/1202093639417968/f
+   */
+  line-height: 1.125rem;
+  color: var(--token-color-surface-faint);
+
+  /**
+   * Note: !important needed, as generic <p> styles leak from DevDotContent.
+   * Task to fix:https://app.asana.com/0/1201987349274776/1201653123138729/f
+   */
+  margin: 4px 0 0 0 !important;
```

Same comment here as the other `!important`, I'm not seeing any styles leak in from `DevDotContent`. They're coming from the user agent, then the `hds-typography-body-100` class, then this local `description` class. Since the `hds-typography-body-100` clears out the `margin`, you should be able to just set `margin-top: 4px` here.

```suggestion
  margin-top: 4px;
```

---

As in `.title` - thanks for this, have cleaned this up! ✅